packages/components-js/tests/vrt/puppeteer/specs/component-high-contrast-mode.vrt.ts

```diff
@@ -65,34 +66,29 @@ const scenarioModalOrBanner = async (page: Page) => {
   await page.mouse.click(0, 0); // click top left corner of the page to remove focus on modal
 };
 
+const scenarioSelectWrapper = async (page: Page) => {
+  await page.click('#open-options');
+};
+
+const highContrastTest = async (component: Component, theme: Theme) => {
```

Comment:
Even so values are identical, we shouldn't use our `Theme` type here since it has nothing to do with the scheme that the value is used for.
See: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme